engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/definition/validator/JsonValidatorExtractor.scala

```diff
@@ -0,0 +1,14 @@
+package pl.touk.nussknacker.engine.definition.validator
+
+import pl.touk.nussknacker.engine.api.definition.{JsonValidator, ParameterValidator}
+
+object JsonValidatorExtractor extends ValidatorExtractor {
+
+  override def extract(params: ValidatorExtractorParameters): Option[ParameterValidator] = {
+    params.extractedEditor match {
+      // TODO add json editor case
```

Comment:
Why TODO? Also I think that you only need to add new case to `AdditionalPropertyEditorValidatorDeterminer` and it should work.

Reply:
I tried to add new case only to `AdditionalPropertyEditorValidatorDeterminer` but it didnt work. `FixedValuesParameterEditor` has case in `AdditionalPropertyEditorValidatorDeterminer` but also in its own extractor, so I did the same.

Reply:
So why this class even exists? Why you've added extractor to the chain that extract nothing and has TODO inside? What is the purpose?

Reply:
@arkadius is right. Please at first merge it: https://github.com/TouK/nussknacker/pull/988. After that uncomment validator extractor.